### DISCUSSION TITLE
More mem places

### DISCUSSION THREAD

**Discussion by AustriaMV** [PE5PVB/TEF6686_ESP32#145] [Ideas]
If 30 mem slots are used, expansion to 100 storage options

**Comment by PE5PVB** [comment]
Isn't 30 enough?

**Reply by HyperDX** [reply]
Exactly. Has anyone used up all the memory presets yet? Is that so urgent? Remember that expanding memory storage reduces the amount of free space for new functions. :)

**Comment by AustriaMV** [comment]
It is easy to fill the memory. First a few Austrian programs, then some from Bavaria. Than the DX Stations from Saxony, Saxony, Anhalt, Thuringia, Berlin, Poland, Czech Republic, Hungary, Croatia. At the end there are a few SW and MW programs. On the road in my area, I often only test DX stations. Eventually, suppress empty unused memory.  SW Station need also som mem places because of missing scan option.
What useful functions are missing?

**Comment by PE5PVB** [comment]
I have extended the number of memory channels to 99.
